lib/components/ComboList.js

```diff
@@ -131,7 +131,7 @@ function onItemsChange(msg, data) {
 
 function MLComboList_get() {
     var value = this.model.get();
-    return typeof value == 'object' ? _.clone(value) : value;
+    return (value && typeof value === 'object') ? _.clone(value) : value;
```

Comment:
You love the `(` and `)` chars

Reply:
You know, I tend to like them sometimes for readability. I know it isn't required, but I find that easier to read.... Your milage may vary.

Reply:
I'm merging this.